### PR TITLE
[Kobo/Cervantes] Prefer using the ioctl over sysfs when setting the FL.

### PR DESCRIPTION
**Pull request by NiLuJe**
It's much lower latency (re #5373).

We can do so on NL devices with a mixer.

### CODE REVIEW

File: frontend/device/kobo/powerd.lua [hunk 1]
```diff
@@ -128,6 +128,14 @@ function KoboPowerD:init()
             -- Does this device's NaturalLight use a custom scale?
             self.fl_warmth_min = self.device.frontlight_settings["nl_min"] or self.fl_warmth_min
             self.fl_warmth_max = self.device.frontlight_settings["nl_max"] or self.fl_warmth_max
+            -- If this device has a mixer, we can use the ioctl for brightness control, as it's much lower latency.
+            if self.device:hasNaturalLightMixer() then
+                local kobolight = require("ffi/kobolight")
+                local ok, light = pcall(kobolight.open)
+                if ok then
+                    self.device.frontlight_settings["frontlight_ioctl"] = light
```
Comment (Frenzie) [inline frontend/device/kobo/powerd.lua]: Or just a dot?

**Comment by pazos** [comment]
@NiLuJe: could you please add the same to cervantes?.

File: frontend/device/kobo/powerd.lua (hunk 1 above)
Comment (NiLuJe) [reply frontend/device/kobo/powerd.lua]: @Frenzie: I'm not sure I follow ;).

Comment (Frenzie) [reply frontend/device/kobo/powerd.lua]: As in self.device.frontlight_settings.frontlight_ioctl is the same thing.

**Pull request merged**